lib/change_health/models/claim/submission/other_subscriber_information.rb

```diff
@@ -0,0 +1,17 @@
+module ChangeHealth
+  module Models
+    module Claim
+      class OtherSubscriberInformation < Model
+        property :benefitsAssignmentCertificationIndicator, from: :benefits_assignment_certification_indicator
+        property :claimFilingIndicatorCode, from: :claim_filing_indicator_code
+        property :claimLevelAdjustments, from: :claim_level_adjustments
+        property :individualRelationshipCode, from: :individual_relationship_code
+        property :otherPayerName, from: :other_payer_name
+        property :otherSubscriberName, from: :other_subscriber_name
+        property :payerPaidAmount, from: :payer_paid_amount
+        property :paymentResponsibilityLevelCode, from: :payment_responsibility_level_code
+        property :releaseOfInformationCode, from: :release_information_code
```

Comment:
Double-checking if you want:
```suggestion
        property :releaseOfInformationCode, from: :release_of_information_code
```